nvidia-device-plugin.yaml

```diff
@@ -38,6 +41,7 @@ pipeline:
 
 update:
   enabled: true
+  manual: false
```

Comment:
Why this change? Just curious

Reply:
This is a default value, honestly I don't see a problem to keep it. But I could change my script.

Reply:
My code is also re-ordering the properties according to the initial yaml structure.

Reply:
AFAIK we normally don't have `manual: false` in these files

Reply:
@luhring Almost all the py3- files set it to false. https://github.com/wolfi-dev/os/pull/6863/files#diff-5fbae266c308c5a4453523df15a67dabe1e6a8e8770be85ea47632df4534be66R57 We have many more files with this property and values true | false.